src/main/scala/uk/gov/nationalarchives/omega/api/messages/IncomingMessageType.scala

```diff
@@ -40,6 +40,10 @@ object IncomingMessageType extends Enum[IncomingMessageType] {
   case object OSLISAGT001 extends IncomingMessageType {
     override val entryName = "OSLISAGT001"
   }
+
+  case object OSGEFREC001 extends IncomingMessageType {
```

Comment:
Should these incoming message types perhaps have some light Scaladoc describing what they are?

Reply:
Added.